bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/aQGC_WZA_LO/WZA_WToLNu_LOaQGC_run_card.dat

```diff
@@ -0,0 +1,267 @@
+  True = clusinfo         ! include clustering tag in output
+  average =  event_norm       ! average/sum. Normalization of the weight in the LHEF
+
+#*********************************************************************
+# Matching parameter (MLM only)
+#*********************************************************************
+ 0 = ickkw            ! 0 no matching, 1 MLM
+ 1.0 = alpsfact         ! scale factor for QCD emission vx
+ False = chcluster        ! cluster only according to channel diag
+ 4 = asrwgtflavor     ! highest quark flavor for a_s reweight
+ True  = auto_ptj_mjj  ! Automatic setting of ptj and mjj if xqcut >0
+                                   ! (turn off for VBF and single top processes) 
+ 0.0   = xqcut   ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# handling of the helicities:
+#  0: sum over all helicities
+#  1: importance sampling over helicities
+#*********************************************************************
+   0  = nhel          ! using helicities importance sampling or not.
+#*********************************************************************
+# Generation bias, check the wiki page below for more information:   *
+#  'cp3.irmp.ucl.ac.be/projects/madgraph/wiki/LOEventGenerationBias' *
+#*********************************************************************
+ None = bias_module  ! Bias type of bias, [None, ptj_bias, -custom_folder-]
+ {} = bias_parameters ! Specifies the parameters of the module.
+#
+#*******************************                                                 
+# Parton level cuts definition *
+#*******************************                                     
+#                                                                    
+#
+#*********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma) ! Define on/off-shell for "$" and decay  
+#*********************************************************************
+  30  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#*********************************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*********************************************************************
+   False  = cut_decays    ! Cut decay products 
+#*********************************************************************
+# Standard Cuts                                                      *
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+ 15  = ptj       ! minimum pt for the jets 
+ 0.0  = ptb       ! minimum pt for the b 
+ 60.0  = pta       ! minimum pt for the photons 
```

Comment:
Isn't this quite high, since you only require pt>20 for your photons in the analysis?

Reply:
> Isn't this quite high, since you only require pt>20 for your photons in the analysis?

In aQGC study, we require photon pt > 60 GeV.
21 page in https://indico.cern.ch/event/1199245/contributions/5042107/attachments/2517158/4327804/22.09.27_WZG_SMPVV_v3.pdf

Reply:
Hi @ico1036 , if, as far as I understand, you require pt(gamma) > 60 GeV at the analysis-level, you have to decrease this threshold at the LHE-level, otherwise there'll be too much overlap between the generated and reconstructed phase space. I would decrease that cut down to at least 40 GeV, if the aQGC analysis is the only use-case foreseen for this sample (if it's not, then you may consider to go further down). I'll let @Dominic-Stafford @menglu21 @Saptaparna comment about this, please correct me if I misunderstood

Reply:
> Hi @ico1036 , if, as far as I understand, you require pt(gamma) > 60 GeV at the analysis-level, you have to decrease this threshold at the LHE-level, otherwise there'll be too much overlap between the generated and reconstructed phase space. I would decrease that cut down to at least 40 GeV, if the aQGC analysis is the only use-case foreseen for this sample (if it's not, then you may consider to go further down). I'll let @Dominic-Stafford @menglu21 @Saptaparna comment about this, please correct me if I misunderstood

Hi @mlizzo  sorry I made a mistake. I modified photon PT cut 60->40 GeV.
Thank you very much for comment.